source/concepts.tex

```diff
@@ -269,21 +269,21 @@
 features. These concepts express relationships between types, type
 classifications, and fundamental type properties.
 
-\rSec2[concept.same]{Concept \libconcept{same_to}}
+\rSec2[concept.same]{Concept \libconcept{same_as}}
 
-\indexlibrary{\idxcode{same_to}}%
+\indexlibrary{\idxcode{same_as}}%
 \begin{itemdecl}
 template<class T, class U>
   concept @\placeholdernc{same-impl}@ = is_same_v<T, U>;    // \expos
 
 template<class T, class U>
-  concept same_to = @\placeholdernc{same-impl}@<T, U> && @\placeholdernc{same-impl}@<U, T>;
+  concept same_as = @\placeholdernc{same-impl}@<T, U> && @\placeholdernc{same-impl}@<U, T>;
```

Comment:
It seems appropriate to rename _`same-impl`_ to _`same-as-impl`_.

Reply:
Combined this with a couple other placeholder name changes mentioned in #3015 as a separate commit.